backend-project/small_eod/channels/models.py

```diff
@@ -1,14 +1,63 @@
 from django.db import models
 from ..generic.models import TimestampUserLogModel
+from django.utils.translation import ugettext_lazy as _
 
 
 class Channel(TimestampUserLogModel):
-    name = models.CharField(max_length=25)
-    city = models.BooleanField(default=False)
-    email = models.BooleanField(default=False)
-    epuap = models.BooleanField(default=False)
-    street = models.BooleanField(default=False)
-    flat_no = models.BooleanField(default=False)
-    house_no = models.BooleanField(default=False)
-    postal_code = models.BooleanField(default=False)
-    voivodeship = models.BooleanField(default=False)
+    name = models.CharField(
+        max_length=25,
+        verbose_name=_("Name"),
+        help_text=_("Channel's name. Name cannot be longer than 25 characters."),
```

Comment:
Podobnie jak w poprzednim PR. Proponuje w "help_text" nie umieszczać informacji o długości, bo ona jest dostępna w osobnym polu dokumentacji API. Jeżeli front-end będzie chciał takie informacje prezentować to wygeneruje na podstawie informacji o długości.

Reply:
Chyba przeczyłam w poprzednim PR info o tym, żeby tego nie umieszczać. Sorry, już poprawiam.